const { log, logerror } = require("../lib/logger");

const serveFailureLogin = (req, res) => {
  try {
    res.render("pages/failureLogin");
    log.info(`${req.method} en ${req.originalUrl}`);
  } catch (err) {
    log.error(`Error obteniendo p치gina fallo de autenticaci칩n ${err}`);
    logerror.error(`Error obteniendo p치gina fallo de autenticaci칩n${err}`);
    res.status(500).json({ message: err.message });
  }
};

module.exports = {
  serveFailureLogin,
};
